mmv1/third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -125,17 +126,22 @@ func isSubnetworkUnreadyError(err error) (bool, string) {
 	return false, ""
 }
 
+var QuotaRegex = regexp.MustCompile(`Quota exceeded for quota metric '(?P<Metric>.*)' and limit '(?P<Limit>.* per minute)' of service 'compute.googleapis.com'`)
+
 // GCE (and possibly other APIs) incorrectly return a 403 rather than a 429 on
 // rate limits.
-func is403ReadRequestsForMinuteError(err error) (bool, string) {
+func isGCE403QuotaExceededPerMinuteError(err error) (bool, string) {
```

Comment:
This is replacing an existing general rule, so let's apply it to all services and not look for GCE

Reply:
Fair enough, will do that.

Reply:
Done